src/common/extension.scss

```diff
@@ -42,6 +42,9 @@ a {
 .secondary-bg {
     background:$nav-bg-color;
 }
+.submenu-bg {
+    background:$submenu-bg;
+}
```

Comment:
I propose to avoid the extraction of styles that used only in a single component to common styles. Defining `submenu-bg` right in `Account.vue` will make easier to track usage of `submenu-bg`.

Reply:
@vmangelovv Can you change this?